error CS0104: 'Contract' is an ambiguous reference

I'm trying to get this up and runny in Unity. I'm getting the following compiler error:
```
error CS0104: 'Contract' is an ambiguous reference between 'LiquidState.Common.Contract' and 'System.Diagnostics.Contracts.Contract'
```

Removing the `using System.Diagnostics.Contracts;` at the head of `StateConfigurationMethodHelper.cs` seems to fix this.

--- a/LiquidState/Synchronous/Core/StateConfigurationMethodHelper.cs
+++ b/LiquidState/Synchronous/Core/StateConfigurationMethodHelper.cs
@@ -4,7 +4,6 @@
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
 using System;
-using System.Diagnostics.Contracts;
 using LiquidState.Core;
 using LiquidState.Common;
 

